analyzer-common/src/main/java/cloud/erda/analyzer/common/constant/Constants.java

```diff
@@ -37,6 +37,10 @@
 
     public static final String TOPIC_METRICS_TEMP = "kafka.topic.metrics_temp";
 
+    //由monitor将记录存入mysql的topic
+    public static final String TOPIC_RECORD_ALERT = "kafka.topic.alert.record";
+    public static final String TOPIC_RECORD_NOTIFY = "kafka.topic.notify.record";
```

Comment:
What is the name of the topic `TOPIC_RECORD_NOTIFY`？

Reply:
kafka.topic.alert.record=spot-alert-record
kafka.topic.notify.record=spot-notify-record